site/docs/reference/Connectors/capture-connectors/gcs.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 4
+---
+# Google Cloud Storage
+
+This connector captures data from an Google Cloud Storage (GCS) bucket.
+
+`ghcr.io/estuary/source-gcs:dev` provides the latest connector image when using the Flow GitOps environment. You can also follow the link in your browser to see past image versions.
```

Comment:
should we actually link the image, since we mention they can follow the link?
```suggestion
[`ghcr.io/estuary/source-gcs:dev`](https://ghcr.io/estuary/source-gcs:dev) provides the latest connector image when using the Flow GitOps environment. You can also follow the link in your browser to see past image versions.
```

Reply:
Probably. The idea was that people are much more likely to be copying it into their console than following it, but I suppose it definitely doesn't hurt, especially since the displayed text doesn't change!